.travis.yml

```diff
@@ -22,4 +22,4 @@ install:
 
 script:
   - gocov test -v ./... > coverage.json
-  - ${HOME}/gopath/bin/goveralls  -gocovdata=coverage.json -service=travis-ci -repotoken=${COVERALLS_TOKEN}
+#  - ${HOME}/gopath/bin/goveralls  -gocovdata=coverage.json -service=travis-ci -repotoken=${COVERALLS_TOKEN}
```

Comment:
$ ${HOME}/gopath/bin/goveralls  -gocovdata=coverage.json -service=travis-ci -repotoken=${COVERALLS_TOKEN}
Bad response status from coveralls: 422 - {"message":"Couldn't find a repository matching this job.","error":true}
The command "${HOME}/gopath/bin/goveralls  -gocovdata=coverage.json -service=travis-ci -repotoken=${COVERALLS_TOKEN}" exited with 1.